src/annotation/annotations_app/models/base.py

```diff
@@ -50,34 +50,38 @@ class User(db.Model):
     is_superuser = db.Column(Boolean, default=False, nullable=False)
 
 
-images_to_collections_table = Table(
-    'images_to_collections_table',
-    db.metadata,
-    Column('image_id', ForeignKey('images.id'), primary_key=True),
-    Column('collection_id', ForeignKey('imagecollections.id'), primary_key=True)
-)
+def generate_short_id(len: int = 15):
+    """
+    Generates short random value, useful for unique IDs but shorter and more readable than UUID
+    """
```

Comment:
OK, but UUID is fine too. Maybe a constraint limiting minimum length would be safer.